src/plugins/plot/res/templates/plot-options-browse.html

```diff
@@ -56,7 +56,7 @@ <h2 title="Plot series display properties in this object">Plot Series</h2>
                     <div class="grid-cell label"
                          title="Whether markers are displayed, and their size.">Markers</div>
                     <div class="grid-cell value">
-                        {{series.get('markers') ? "Enabled: " + series.get('markerSize') + "px" : "Disabled"}}
+                        {{ series.get('markers') ? series.get('markerShape')[0].toUpperCase() + series.get('markerShape').substring(1) + ": " + series.get('markerSize') + "px" : "Disabled" }}
```

Comment:
I see this is one of those cases where a ternary is useful, but it does challenge readability. Could this be a computed property in which a standard if condition was used?

Reply:
Unfortunately, this is an angular template. If it were a Vue SFC I'd totally agree.

Reply:
could we have a helper method to get us what we need here?

Reply:
Alright. The refactors have it.

Moved the logic into a function in series.